src/lib/viewers/BaseViewer.js

```diff
@@ -214,6 +214,10 @@ class BaseViewer extends EventEmitter {
             });
         }
 
+        if (this.options.enableAnnotationsDiscoverability) {
+            this.containerEl.classList.add('bp-annotations-discoverable');
```

Comment:
Does it make sense to move these classes to constants since they're being used in a few places?

Reply:
+1